lib/internal/util.js

```diff
@@ -789,6 +794,24 @@ function setupCoverageHooks(dir) {
   return coverageDirectory;
 }
 
+// This would better be placed in internal/worker/io.js, but that doesn't work
+// because Buffer needs this and that would introduce a cyclic dependency.
+// Mark the object as untransferable. If object occurs in the transfer list of
+// a port.postMessage() call, it is ignored.
+function markAsUntransferable(obj) {
+  if ((typeof obj !== 'object' && typeof obj !== 'function') || obj === null)
```

Comment:
Nit: We can revert the order of this function for small optimization since typeof is slower than null equality?

Reply:
Should this function also check for undefined values? Instead of triple equal, maybe we can use double equal operator?